language/unit_tests/test_client.py

```diff
@@ -26,19 +26,25 @@ def _make_one(self, *args, **kw):
         return self._get_target_class()(*args, **kw)
 
     def test_ctor(self):
+        import mock
+        from oauth2client.client import GoogleCredentials
         from google.cloud.language.connection import Connection
 
-        creds = _Credentials()
+        creds = mock.Mock(spec=GoogleCredentials)
+        creds.create_scoped_required.return_value = False
```

Comment:
Ignore the kwargs bit for the return-value parts:  the helper function / method would just take the desired 'create_scope_required` value and configure it appropriately.